pkg/translator/api/ingress.go

```diff
@@ -61,7 +82,17 @@ func (o *IngressEdgeLBPoolSpec) Validate(obj *extsv1beta1.Ingress) error {
 	}
 	// Validate that the HTTP port is valid.
 	if err := validation.IsValidPortNum(int(*o.Frontends.HTTP.Port)); err != nil {
-		return fmt.Errorf("%d is not a valid port number", *o.Frontends.HTTP.Port)
+		return fmt.Errorf("%d is not a valid HTTP port number", *o.Frontends.HTTP.Port)
+	}
+	// Validate that the HTTP mode is valid.
+	if err := isValidHTTPMode(*o.Frontends.HTTP.Mode); err != nil {
+		return fmt.Errorf("%s is not a valid HTTP mode", *o.Frontends.HTTP.Mode)
```

Comment:
Field path and what are the valid modes would be great.

Reply:
Done.